daprdocs/content/en/operations/hosting/kubernetes/kubernetes-production.md

```diff
@@ -260,6 +260,20 @@ Verify your production-ready deployment includes the following settings:
 
 1. Dapr supports and is enabled to **scope components for certain applications**. This is not a required practice. [Learn more about component scopes]({{< ref "component-scopes.md" >}}).
 
+## Recommended Placement service configuration
+
+The [Placement service]({{< ref "placement.md" >}}) is a critical component in Dapr, responsible for actor placement table dissemination. It's recommended to configure the Placement service with the following values:
+
+1. **High availability**. Ensure that the Placement service is highly available and can survive individual node failures. Helm chart value: `dapr_placement.ha=true`
```

Comment:
```suggestion
1. **High availability**. Ensure that the placement service is highly available (3 replicas) and can survive individual node failures. Helm chart value: `dapr_placement.ha=true`
```

Reply:
Placement service is a name so it is Placement service rather than placement service.